File: isegoria/src/main/java/com/eulersbridge/isegoria/models/ElectionInstitution.java
package com.eulersbridge.isegoria.models;

import com.squareup.moshi.Json;

class ElectionInstitution {

    @Json(name = "institutionId")
    public long id;

    public String name;
    public String campus;
    public String state;
    public String country;
}

File: isegoria/src/main/kotlin/com/eulersbridge/isegoria/network/api/model/ElectionInstitution.kt
package com.eulersbridge.isegoria.network.api.model

import com.squareup.moshi.Json

data class ElectionInstitution (
    @field:Json(name = "institutionId")
    var id: Long = 0,

    val name: String?,
    val campus: String?,
    val state: String?,
    val country: String?
)
